api/src/main/resources/db/changelog/db.changelog-fees-managed-v5.yaml

```diff
@@ -122,3 +122,15 @@ databaseChangeLog:
         constraintName: fk_fee_banded_fee
         referencedColumnNames: id
         referencedTableName: fee
+
+
+
+- changeSet:
+    id: 1530529796319
+    author: sachi
+    changes:
+      - sql:
+          sql: >
+            INSERT INTO service_type (name,creation_time,last_updated)
+            VALUES ('other',current_timestamp,current_timestamp)
+
```

Comment:
Please check if its good idea to add 
ON CONFLICT (name)
DO NOTHING;
i.e. it shouldn't fail if this value already exists for whatsoever reason.